packages/suite/src/components/wallet/AmountComponent.tsx

```diff
@@ -0,0 +1,44 @@
+import { isNftTokenTransfer, formatAmount, getTxOperation } from '@suite-common/wallet-utils';
+import { TokenTransfer } from '@trezor/connect';
+
+import { FormattedNftAmount } from 'src/components/suite/FormattedNftAmount';
+import { FormattedCryptoAmount } from 'src/components/suite/FormattedCryptoAmount';
+
+type AmountComponentProps = {
+    transfer: TokenTransfer;
+    withLink?: boolean;
+    withSign?: boolean;
+    alignMultitoken?: 'flex-end' | 'flex-start';
+};
+
+export const AmountComponent = ({
+    transfer,
+    withLink = false,
+    withSign = false,
+    alignMultitoken,
+}: AmountComponentProps): React.ReactNode => {
+    const operation = getTxOperation(transfer.type);
+
+    if (isNftTokenTransfer(transfer)) {
+        return (
+            <FormattedNftAmount
```

Comment:
I am not sure how the hidden placeholder works but for normal amounts, it replaces the amount by XXX, for nfts, it does not do that. Probably it was there before so just create issue if it is hard to fix
![Screenshot 2024-11-29 at 17 35 08](https://github.com/user-attachments/assets/f131e52d-194b-4954-b200-f022d02b1419)

Reply:
I will create an issue, now there's only a method to redact a raw string or numebr, but we have a component because of BlurUrls in the name

Reply:
Issue created